lib/language.test.js

```diff
@@ -47,13 +50,14 @@ describe('languageHandler', () => {
   })
 
   it('should use accept-language if present', () => {
```

Comment:
why do we expect 'en', should we make that more clear(I think it's better to use the variable as a part of accept-language)?